ts/components/conversation/UpdateGroupDialog.tsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
```

Comment:
Feels like this should be able to be combined with CreateGroupDialog but would need more intricate understanding of the issues to know how easy that would be

Reply:
I thought about it, but considering there are enough differences (at least there were, at one or another point of me experimenting with the design), I thought keeping them separate is both cleaner and faster to implement.